clients/src/main/java/org/apache/kafka/clients/ClientUtils.java

```diff
@@ -228,6 +229,10 @@ public static NetworkClient createNetworkClient(AbstractConfig config,
                     metricsGroupPrefix,
                     channelBuilder,
                     logContext);
+            bootstrapConfig = new NetworkClient.BootstrapConfiguration(
```

Comment:
The bootstrap server addresses are stored in metadata per KIP-899 but metadata.rebootstrap() is called within metadata.maybeUpdate().  So I don't know if it is easy to combine both config.

Reply:
Could you elaborate on that? I am not sure what configs you are talking about, I am only seeing one.

Reply:
KIP-899 allows clients to rebootstrap.  so it stores the bootstrap server addresses in the metadata. check `Metadata.java`